### DISCUSSION TITLE
APOE: Error from Osaka

### DISCUSSION THREAD

**Discussion by krootesmurdy** [trendscenter/coinstac#1463] [General]
Error from this morning: "TypeError: Cannot read properties of undefined (reading 'find')"

for context, they are trying to map csv columns for preprocessing pipeline

**Comment by krootesmurdy** [comment]
we rearranged the columns, corrected the image files, and moved everything to the same folder. Still getting the error:

"TypeError: Cannot read properties of undefined (reading 'find')"

**Comment by krootesmurdy** [comment]
Solution: CLEAR LOCAL DATA

Unclear as to what the actual problem is but removing the local data allows the site to re-map without issue.

**Reply by krootesmurdy** [reply]
the site used CLEAR LOCAL DATA and then tried to re-map but still got the same error message "Cannot read properties of undefined"

**Reply by rssk** [reply]
Hey Kelly just to be specific here, "TypeError: Cannot read properties of undefined (reading 'find')" is the error?

**Reply by krootesmurdy** [reply]
yes. csv files were checked.
